doc/new-guidelines.md

```diff
@@ -80,6 +80,63 @@ This mistake mostly happens with functional components
 
 it's important to name functional components with `f-` prefix
 
+### Component props and api scheme to match Figma as closely as possible
+
+Ideally, the prop names for components (particularly in quo2 Design System) should match the Figma properties as best 
+as possible. This makes it easier for the developer using that component to then configure it correctly for the screen it is
+being used on and avoids unnecessary overwrites and adjustments being made.
+
+Some examples include
+
+#### Avoid unnecessarily grouping categories to reduce the number of props
+For example in Figma if there is component and it has variants the following variants
```

Comment:
Typo fix: For example in Figma if there is a component and it has the following variants